pyproject.toml

```
@@ -83,7 +83,7 @@ dependencies = [
   "torch>=2.6.0",
   "sentencepiece>=0.1.99",
   "markdown>=3.4.3",
-  "numpy>=1.26.4,<2.0",
+  "numpy>=1.21.0,<=1.24.0",
```

Why is this restricting to a an old `1.x` version?
```suggestion
  "numpy>=1.26.4,<2.0",
```